src/main.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import App from '../src/App/App';
+import './index.css';
```

Comment:
imports of css are the last in list of imports

Reply:
corrected